CA_DataUploaderLib/IOconf/IOconfFile.cs

```diff
@@ -106,6 +102,20 @@ public static ConnectionInfo GetConnectionInfo()
             }
         }
 
+        public static string GetLoopName()
+        {
+            return ((IOconfLoopName)Table.Single(x => x.GetType() == typeof(IOconfLoopName))).Name;
+        }
+
+        public static int GetVectorUploadDelay()
+        {
+            return ((IOconfSamplingRates)Table.Single(x => x.GetType() == typeof(IOconfSamplingRates))).VectorUploadDelay;
```

Comment:
suggest to have a default, like: 
```
public static int GetVectorUploadDelay() => 
   (Table.OfType<IOConfSamplingRates>().SingleOrDefault() ?? IOConfSamplingRates.Default).VectorUploadDelay
```
Where this is a static readonly field with the default values close to current behavior (1Hz for upload and 5 Hz for sampling).

Reply:
@copenhagenatomics I have the impression the commit did not make it in?